src/viam/sdk/services/motion/client.cpp

```diff
@@ -135,6 +135,71 @@ pose_in_frame MotionClient::get_pose(
     return pose_in_frame::from_proto(response.pose());
 }
 
+void MotionClient::stop_plan(const Name& name, const AttributeMap& extra) {
+    service::motion::v1::StopPlanRequest request;
+    service::motion::v1::StopPlanResponse response;
+    grpc::ClientContext ctx;
+    set_client_ctx_authority(ctx);
+
+    *request.mutable_name() = this->name();
+    *request.mutable_extra() = map_to_struct(std::move(extra));
```

Comment:
A [different line in this patch](https://github.com/viamrobotics/viam-cpp-sdk/pull/181/files#diff-99d8513b064c421efbffb53e9b3d68e08962e998f22ceb61037f9c8fd9a0d766R200) did not `std::move`.

I think* a `std::move(const reference)` just turns into a copy, so this seems safe to me. Just not useful.

Reply:
Thanks for the catch! will remove the superfluous moves.